doc/man7/EVP_PKEY-SM2.pod

```diff
@@ -15,6 +15,23 @@ both signature and encryption schemes via the EVP interface.
 When doing the B<SM2> signature algorithm, it requires a distinguishing identifier
 to form the message prefix which is hashed before the real message is hashed.
 
+=head2 Common SM2 parameters
+
+SM2 uses the same parameters as defined in L<EVP_PKEY-EC(7)/Common EC parameters>.
```

Comment:
d/same/

Reply:
Changed to uses the parameters defined ..